fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -231,6 +237,17 @@ protected ResponseBuilder addHeaders(ResponseBuilder rb, Resource resource) {
                 .lastModified(Date.from(resource.getMeta().getLastUpdated().getValue().toInstant()));
     }
 
+    /**
+     * Adds the Etag header to the specified response object.
+     * @param rb
+     * @param resource
```

Comment:
Extra "resource" param does not exist in method.

Reply:
Since this is the only outstanding comment, I'm going to:
1. add the `ci-skip` label to the PR
2. make a single commit to remove this one line
3. press merge on the PR

```suggestion
```